addons/captives/functions/fnc_setHandcuffed.sqf

```diff
@@ -17,7 +18,7 @@
  * Public: No
  */
 
-params ["_unit", "_state", ["_caller", objNull]];
+params ["_unit", "_state", ["_caller", objNull], "_selectedCuffs"];
```

Comment:
```suggestion
params ["_unit", "_state", ["_caller", objNull], "_selectedRestraint"];
```

Should use restraint instead of cuffs throughout the code, although we obviously can't change the events, unfortunately